### PR TITLE
fix 4157: allow audience populationPercent to be a decimal number

### PR DESCRIPTION
**Pull request by lmorchard**

### CODE REVIEW

**Comment by lmorchard** [comment]
I haven't played with `type="number"` inputs a bunch, but this seems to fix the issue?

File: app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx [hunk 1]
```diff
@@ -234,6 +234,7 @@ export const FormAudience = ({
                 type="number"
                 min="0"
                 max="100"
+                step="0.01"
```
Comment (jaredlockhart) [inline app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx:237]: populationPercent allows 4 decimal places:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/models/nimbus.py#L47-L49

**Review by jaredlockhart — CHANGES_REQUESTED** [review]

File: app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx (hunk 1 above)
Comment (lmorchard) [reply app/experimenter/nimbus-ui/src/components/FormAudience/index.tsx:237]: I don't think this restricts the number of decimal places, just indicates that it's not an integer and provides a step value for the up/down arrows

**Review by jaredlockhart — APPROVED** [review]
👍 🐱

**Pull request merged**